### PR TITLE
rm choo-log

### PR DESCRIPTION
**Pull request by yoshuawuyts**
Removes `choo-log` from the install step. Depends on https://github.com/choojs/choo-devtools/pull/14, and related to https://github.com/choojs/discuss/issues/2. Thanks!

Sibling PR – https://github.com/choojs/create-choo-app/pull/28

### CODE REVIEW

File: bin.js [hunk 1]
```diff
@@ -100,7 +99,7 @@ function create (dir, argv) {
     },
     function (done) {
       var pkgs = [
-        'bankai@next',
+        'bankai',
```
Comment (lrlna) [inline bin.js:102]: is bankai not on `rc7`, etc anymore, or will this resolve to that? (might be a silly question, sorry if it is)

Comment (yoshuawuyts) [reply bin.js:102]: Oh yeah, so `npm install bankai` installs the latest RC out of the box right now. We messed up somewhere, but ah well — at least this code does what it's supposed to do this way (:

Comment (lrlna) [reply bin.js:102]: oh cool! i was asking cause couple of weeks ago it would just install 9.0.0, but i guess something got fixed!

**Review by lrlna — APPROVED** [review]

**Comment by yoshuawuyts** [comment]
📦 `v1.7.0`

**Pull request merged**